src/components/hermes-skills/useSkillDetail.ts

```
@@ -256,10 +274,19 @@ export function useSkillDetail(inspectId: string | null, fromInstalled = false):
   const heldFor = detailKey(inspectId, fromInstalled);
   const stale = !held || held.key !== heldFor || held.skill.id !== inspectId;
   const fromThisRun = (a: { key: string; run: number }) => a.key === heldFor && a.run === epoch.current;
+  const failure = errorState && fromThisRun(errorState) ? errorState : null;
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/src/components/hermes-skills/useSkillDetail.ts:277:33
  275 |   const stale = !held || held.key !== heldFor || held.skill.id !== inspectId;
  276 |   const fromThisRun = (a: { key: string; run: number }) => a.key === heldFor && a.run === epoch.current;
> 277 |   const failure = errorState && fromThisRun(errorState) ? errorState : null;
      |                                 ^^^^^^^^^^^ This function accesses a ref value
  278 |   return {
  279 |     detail: stale ? null : held.skill,
  280 |     // `stale` means "nothing held for this selection", which is true both

(react-hooks/refs)